docs/rules/no-duplicate-imports.md

```diff
@@ -1,6 +1,8 @@
 # Disallow duplicate imports (no-duplicate-imports)
 
-An ES6/ES2015 import can be spread over multiple lines, but this takes up unneeded whitespace. This rules validates that all imports from a single module exists in a single import statement.
+An ES6/ES2015 import can be spread over multiple lines, but this takes up unneeded whitespace.
```

Comment:
Yes, thank you for your patience.

The sentence with “because you can see everything” seems like a such a strong reason that maybe clearer to drop the sentence about “unneeded whitespace” because some readers might feel that is more of a style issue.